src/styles/less/EditPlayers.less

```diff
@@ -110,22 +113,27 @@
                         width: 90%;
 
                         .ant-form-item-label {
+                            width:100%;
                             text-align: start;
                             font-family: 'Open Sans';
                             font-weight: 700;
                             font-size: 0.8rem;
-                            color: #4B5151;
+                            color: #2D2A2B;
                             margin-left: 10px;
                             ::after {
                                 display: none;
                             }
                         }
                         input {
-                            .inputStyle();
+                            // .inputStyle();
+                            border: 0.5px solid #7b8794;
+                            border-radius: 16px;
+                            padding: 7px;
+                            font-size: 16px;
```

Comment:
I'm a big fan of responsive units, especially for font-size. Just a consideration, I'm not familiar with how ant design handles their components.

Reply:
It is going to be interesting to see the change  :)